### PR TITLE
Fixes #1212: CompletedStoryList - Tablet (Portrait) (Lowfi)

### LINKED ISSUE #1212: CompletedStoryList - Tablet (Portrait) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/370479e0-c774-4ca2-9864-bbf9f58d5672/P-Stories-Completed  

Implement low-fi UI for **CompletedStoryList** tablet portrait mode

**Target PR date**: 30 July 2020
**Target completion date**: 3 August 2020

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Implemented low-fi UI for **CompletedStoryList" for portrait mode - tablet
- Fixes #1212 
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by rt4914 — CHANGES_REQUESTED** [review]
LGTM, we will need to write test cases for span count check. similar to `ProfileChooserSpanTest` for this file and all the rest of the earlier files too.

**Comment by BenHenning** [comment]
@MohamedMedhat1998 please update this with the latest develop.

**Comment by MohamedMedhat1998** [comment]
> 
> 
> @MohamedMedhat1998 please update this with the latest develop.

Done

**Comment by MohamedMedhat1998** [comment]
> LGTM, we will need to write test cases for span count check. similar to `ProfileChooserSpanTest` for this file and all the rest of the earlier files too.

@rt4914 Done

File: app/src/main/java/org/oppia/app/completedstorylist/CompletedStoryListFragment.kt [hunk 1]
```diff
@@ -11,6 +11,7 @@ import javax.inject.Inject
 /** Fragment for displaying completed stories. */
 class CompletedStoryListFragment : InjectableFragment() {
   companion object {
+    internal const val COMPLETED_STORY_LIST_FRAGMENT_TAG = "completed_story_list_fragment_tag"
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/completedstorylist/CompletedStoryListFragment.kt]: Nit: 
`internal const val COMPLETED_STORY_LIST_FRAGMENT_TAG  = "COMPLETED_STORY_LIST_FRAGMENT_TAG"`

File: app/src/test/java/org/oppia/app/testing/CompletedStoryListSpanTest.kt [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      completedStoryListFragment?.view?.findViewWithTag<RecyclerView>(
+        activity.resources.getString(
+          R.string.completed_story_list_recyclerview_tag
+        )
+      )
+    return (completedStoryListRecyclerVIew?.layoutManager as GridLayoutManager).spanCount
+  }
+
+  @Test
+  fun testCompletedStoryListFragmentRecyclerViewHasCorrectSpanCount() {
```
Comment (rt4914) [inline app/src/test/java/org/oppia/app/testing/CompletedStoryListSpanTest.kt]: Rename to `testCompletedStoryList_checkRecyclerViewSpanCount_spanIsCorrect`

Comment (rt4914) [reply app/src/test/java/org/oppia/app/testing/CompletedStoryListSpanTest.kt]: Or `Rename to `testCompletedStoryList_checkRecyclerViewSpanCount_hasCorrectSpanCorrect``

Comment (rt4914) [reply app/src/test/java/org/oppia/app/testing/CompletedStoryListSpanTest.kt]: Similar change for other test cases.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Overall looks good, some minor changes suggested.

File: app/src/main/java/org/oppia/app/completedstorylist/CompletedStoryListFragment.kt (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/main/java/org/oppia/app/completedstorylist/CompletedStoryListFragment.kt]: Done

File: app/src/test/java/org/oppia/app/testing/CompletedStoryListSpanTest.kt (hunk 2 above)
Comment (MohamedMedhat1998) [reply app/src/test/java/org/oppia/app/testing/CompletedStoryListSpanTest.kt]: Done

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**